packages/react/src/auto/AutoForm.ts

```diff
@@ -69,6 +74,29 @@ const useValidationResolver = (metadata: ModelWithOneActionMetadata | GlobalActi
   }, [metadata, pathsToValidate]);
 };
 
+const isMetadataForUpsertAction = (metadata: ModelWithOneActionMetadata | GlobalActionMetadata | undefined | null) => {
```

Comment:
I think we have `isUpsertMetaAction` on the meta schema now; should we use that instead of attempting to infer it?

Reply:
I'm cool to do this in a follow up though